app/controllers/api/v1/dependencies_controller.rb

```diff
@@ -40,6 +40,7 @@ def cacheable!
 
   def check_brownout
     current_time = Time.current.utc
+    return if Patterns::JAVA_HTTP_USER_AGENT.match?(request.user_agent)
```

Comment:
this won't work since nothing about the user-agent is included in the `VARY` header

Reply:
Also no need to get current time if this will just immediately return upon matching the user agent.

Reply:
> this won't work since nothing about the user-agent is included in the `VARY` header

Sorry, I don't follow. Doing request locally I can get user agent by `request.headers["HTTP_USER_AGENT"]` or `request.env["HTTP_USER_AGENT"]` and same value is present at `request.user_agent`. This value is also used at webauthn prompt. Should be different value used?

Reply:
@headius fixed

Reply:
@segiddins I found out that method is defined at https://github.com/rack/rack/blob/983b6e3b29a2048a86518c008fc46f4c86105683/lib/rack/request.rb#L161.